### ISSUE TITLE
Uncaught Error: ESPIPE: invalid seek, write

### ISSUE THREAD

**Issue by bruninja** [atom/tabs#197]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.2
**System**: "Fedora release
**Thrown From**: [tabs](https://github.com/atom/tabs) package, v0.82.0

### Stack Trace

Uncaught Error: ESPIPE: invalid seek, write

```
At /usr/share/atom/resources/app.asar/src/pane.js:774

Error: ESPIPE: invalid seek, write
  at Error (native)
  at Object.fs.writeSync (fs.js:657:20)
  at Object.fs.writeFileSync (fs.js:1164:21)
  at Object.fsPlus.writeFileSync (/usr/share/atom/resources/app.asar/node_modules/fs-plus/lib/fs-plus.js:279:17)
  at File.module.exports.File.writeFileSync (/usr/share/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:273:19)
  at File.module.exports.File.writeFileWithPrivilegeEscalationSync (/usr/share/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:371:21)
  at File.module.exports.File.writeSync (/usr/share/atom/resources/app.asar/node_modules/pathwatcher/lib/file.js:345:12)
  at TextBuffer.module.exports.TextBuffer.saveAs (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:992:17)
  at TextBuffer.module.exports.TextBuffer.save (/usr/share/atom/resources/app.asar/node_modules/text-buffer/lib/text-buffer.js:978:19)
  at TextEditor.module.exports.TextEditor.save (/usr/share/atom/resources/app.asar/src/text-editor.js:579:26)
  at Pane.module.exports.Pane.saveItem (/usr/share/atom/resources/app.asar/src/pane.js:535:18)
  at Pane.module.exports.Pane.promptToSaveItem (/usr/share/atom/resources/app.asar/src/pane.js:507:23)
  at Pane.module.exports.Pane.destroyItem (/usr/share/atom/resources/app.asar/src/pane.js:453:18)
  at HTMLDivElement.<anonymous> (/usr/share/atom/resources/app.asar/node_modules/tabs/lib/tab-bar-view.js:236:22)
  at space-pen-ul.jQuery.event.dispatch (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at space-pen-ul.elemData.handle (/usr/share/atom/resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```

### Commands

```
     -4:05.9.0 find-and-replace:show (atom-text-editor.editor.is-focused)
     -4:00.0 core:confirm (atom-text-editor.editor.mini.is-focused)
  2x -3:26.6.0 core:save (atom-text-editor.editor.is-focused)
     -2:45.4.0 core:select-all (atom-text-editor.editor.is-focused)
     -2:44.3.0 core:copy (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {}
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by bruninja** [comment]
This error occurs while update a file by ftp.

**Comment by kevinsawicki** [comment]
This was fixed by https://github.com/atom/atom/pull/7802 and will be included in Atom 1.0.3
